run.py

```diff
@@ -69,7 +69,7 @@ def download_package(client, name, version):
         break
     else:
         for bdist in dists['bdist_wheel']:
-            if WheelFile(bdist['filename']).compatible:
+            if WheelFile(file=bdist['filename']).compatible:
```

Comment:
This is the property I was using https://github.com/pypa/wheel/blob/0.31.1/wheel/install.py#L152-L154
The API seems gone entirely now, so we have to find something else...

Reply:
Looking at the [sources](https://github.com/pypa/wheel/blob/master/wheel/wheelfile.py#L58), it seems we can catch `WheelError` when constructing the object to find out if the wheel is valid.

Reply:
Catching `WheelError` to detect if it is a valid wheel. 👍